azure-pipelines-docs-integration.yml

```diff
@@ -55,6 +55,15 @@ stages:
           - bash: ./scripts/check_for_docs_deps_changes
             name: CheckDocsDependenciesChanges
 
+  - stage: custom_checks
+    pool:
+      vmImage: 'ubuntu-latest'
+    jobs:
+    - job: link_checker
+      condition: or(eq(stageDependencies.scope_check.changes.outputs['CheckDocsChanges.DocsChanged'], true), eq(variables.isMain, true))
```

Comment:
Should we also add `CheckDocsDependenciesChanges` to this condition? See L72 below for reference.

This is a script that determines whether or not source code in GE has changed that might have an impact on our docs under test.

Reply:
I left it out because I didn't think those dependencies could result in broken links

Reply:
You are 100% correct disregard that comment!